nautobot/extras/tests/test_views.py

```diff
@@ -881,7 +882,93 @@ def test_only_enabled_is_listed(self):
         self.assertHttpStatus(response, 200)
         self.assertNotIn("test4", extract_page_body(response.content.decode(response.charset)))
 
-    # TODO: add `def test_non_valid_crontab_syntax(self):`
+    def test_non_valid_crontab_syntax(self):
+        self.add_permissions("extras.view_scheduledjob")
+
+        with self.assertRaises(ValidationError):
+            ScheduledJob.objects.create(
+                enabled=True,
+                name="test10",
+                task="nautobot.extras.jobs.scheduled_job_handler",
+                job_class="local/test_pass/TestPass",
+                interval=JobExecutionType.TYPE_CUSTOM,
+                user=self.user,
+                start_time=timezone.now(),
+                crontab=None,
+            )
```

Comment:
Since this pattern is repeated multiple times with seemingly only the argument to `crontab` changing, can you please move this into a factory method that can be called taking a `crontab` parameter?